website/docs/d/launch_configuration.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: "aws"
+page_title: "AWS: aws_launch_configuration"
+sidebar_current: "docs-aws-datasource-launch-configuration"
+description: |-
+  Provides a Launch Configuration data source.
+---
+
+# aws_launch_configuration
```

Comment:
This PR probably predates when we applied this across the provider, but can you please prepend `Data Source: ` to the header, e.g. `Data Source: aws_launch_configuration`